components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -52,6 +52,9 @@ function InputBox(props: InputBoxProps) {
   }
 
   const querySubmitted = React.useRef(false)
+  const attachmentWrapperRef = React.useRef<HTMLDivElement>(null)
+  const [attachmentWrapperHeight, setAttachmentWrapperHeight] =
+    React.useState(0)
```

Comment:
sorry to be a pain, but do you reckon you could try to get this working with [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries)?

It just makes me uncomfortable when we start measuring stuff with JS

If it ends up being heaps more code, or you reckon its worse I'm happy to land as-is, but it'd be good to see if we can avoid it.